imagersite/imager_images/templates/imager_images/album-detail.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'imagersite/base.html' %}
+
+{% block body %}
+<main class="container">
+  <div class="row">
+    <div class="col-sm-12">
+      <h3>Photos in Album: {{ album.title }}</h3>
+      <ul>
+        {% for p in album.photos.all %}
```

Comment:
A better variable name would eliminate the possibility of confusing `{{ p }}` django tags with `<p>` HTML tags.